frontend/src/components/DetailsPageSpendingField.tsx

```diff
@@ -35,12 +37,18 @@ export default function DetailsPageSpendingField({title, itemID, itemClass, invo
         setAddSpendingMode(false)
     }
 
+    const onDelete = () => {
+        removeSpending(spending)
+            .then(response => spending === response && setSpending(""))
+        saveItem()
+    }
+
     return <div>
-        {spending && editMode ? <Button variant={"danger"}>Ausgabe Löschen</Button>
+        {spending && editMode ? <Button variant={"danger"} onClick={() => onDelete()}>Ausgabe Löschen</Button>
```

Comment:
```suggestion
        {spending && editMode ? <Button variant={"danger"} onClick={onDelete}>Ausgabe Löschen</Button>
```
Sollte so auch gehen oder?